src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const ArticlePage = () => {
+  const { id } = useParams();
+  console.log(useParams());
+  const [article, setArticle] = useState(null);
+
+  useEffect(() => {
+    fetch(`https://my-nc-news-yyln.onrender.com/api/articles/${id}`)
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        console.log(data.article);
+        setArticle(data.article);
+      });
+  }, [id]); // this fetches the article if the id changes
```

Comment:
Nice, but I would recommend looking into axios as a nice alternative to this which can make some of the later requests a lot easier.

Notes:
https://l2c.northcoders.com/courses/sd-notes/front-end#sectionId=axios,step=intro